gsplat/strategy/default.py

```diff
@@ -232,9 +247,18 @@ def _update_state(
             # grads is [C, N, 2]
             sel = info["radii"] > 0.0  # [C, N]
             gs_ids = torch.where(sel)[1]  # [nnz]
+            if len(grads.shape) != 3:
+                grads = grads.unsqueeze(0)
```

Comment:
Why do we need this? The grads should always be in shape of [nnz, 2] right?

Reply:
right, this is for inria-implementation benchmark; will remove this.